doc/template.html

```diff
@@ -9,6 +9,7 @@
   <link rel="stylesheet" href="assets/style.css">
   <link rel="stylesheet" href="assets/hljs.css">
   <link rel="canonical" href="https://nodejs.org/api/__FILENAME__.html">
+  <script src="assets/script.js"></script>
```

Comment:
Why not add to the existing script?

Reply:
@aduh95 asked to move it to a different file 
>maybe you can put this to a separate .js file in doc/api_assets so the linter can autofix that for you.